dwave/system/testing.py

```diff
@@ -400,20 +413,8 @@ def sample(self, bqm, **kwargs):
         num_reads = kwargs.get('num_reads', substitute_kwargs.get('num_reads', 1))
         substitute_kwargs['num_reads'] = num_reads
 
-        if 'initial_state' in kwargs:
```

Comment:
I guess just passing `initial_state` as `initial_states` to the greedy solver (default substitute sampler) would work, but it creates another special case. Ignoring initial states is fine.